api/v6/report_server.thrift

```diff
@@ -262,7 +264,8 @@ service codeCheckerDBAccess {
   // counts will be calculated for all of the available runs.
   // PERMISSION: PRODUCT_ACCESS
   RunReportCounts getRunReportCounts(1: list<i64>    runIds,
-                                     2: ReportFilter reportFilter)
+                                     2: ReportFilter reportFilter,
+                                     3: i64          limit)
```

Comment:
These limit variables could be optional.

Reply:
Do we want to introduce offset too?

Reply:
It seems to me that thrift doesn't support optional function arguments. The compiler gives me the following warning: `optional keyword is ignored in argument lists`. (https://svn.apache.org/viewvc/incubator/thrift/trunk/compiler/cpp/src/thrifty.yy?view=markup&pathrev=665255)